.buildkite/pipeline.yml

```diff
@@ -116,3 +116,11 @@ steps:
     - buildkite-agent artifact download Holdings-contract.md tmp/
         --step "contract doc release"
     - ./scripts/ci/upload-autodoc.sh
+
+notify:
+  - email: "tezos-alerts@serokell.io"
+    if: &failed_scheduled
+      build.state == "failed" && build.source == "schedule"
+  # However, slack app seems to be not working properly:(
```

Comment:
So what? Does it do nothing? Where is the problem exactly? Do use think we should keep it here hoping they will fix it one day?

Reply:
right, let's remove it